vm/src/vm.rs

```diff
@@ -1322,9 +1323,7 @@ impl Default for VirtualMachine {
     }
 }
 
-lazy_static! {
-    static ref REPR_GUARDS: Mutex<HashSet<usize>> = { Mutex::new(HashSet::new()) };
-}
+static REPR_GUARDS: Lazy<Mutex<HashSet<usize>>> = Lazy::new(|| Mutex::new(HashSet::new()));
```

Comment:
Could this be `Lazy::default()`?

Reply:
Probably!

Reply:
Unfortunately I don't think so.  I don't see anything like that documented and throws a compile error

Reply:
Really? It looks like `Default` is implemented for `Lazy<T: Default>` [here](https://docs.rs/once_cell/1.3.1/src/once_cell/lib.rs.html#930-935).

Reply:
Oh, maybe it's not a `const fn`. Could do `Lazy::new(Mutex::default)`, though.

Reply:
Yup, that's it.  But the latter strategy works